subprojects/packagefiles/cpptrace/meson_options.txt

```diff
@@ -0,0 +1,51 @@
+option(
+    'include-system',
+    type: 'boolean',
```

Comment:
This variable is unused. To my knowledge, a meson dependency cannot mandate how its header files should be included, the consumer has to specify that: https://mesonbuild.com/Reference-manual_functions.html#dependency_include_type